src/components/ImportRecordsFlow/__test__/footer.spec.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Footer from '../footer';
+
+describe('<Footer />', () => {
+    it('should have back button disabled when isBackButtonDisabled is set to true', () => {
+        const component = mount(<Footer isBackButtonDisabled />);
+        const prevButton = component.find('button').first();
```

Comment:
Inside Footer, we are using the Button component and here what we need to check is that the disabled prop on the Button (StyledBackButton) component is the same value of isBackButtonDisabled because checking for the button tag element is introspection on the Button component. Remember that Footer does not know implementation details about Button.